installing/acs-installation-platforms.adoc

```diff
@@ -1,14 +1,144 @@
+|Limited ^[4]^
+|Yes
+.3+d|Helm charts (recommended), or roxctl CLI ^[3]^
+
+.3+a| * xref:../installing/installing_other/install-rhacs-other.adoc#install-rhacs-other[High-level overview of installing {product-title-short} on other platforms]
+* xref:../installing/installing_cloud_other/install-rhacs-cloud-other.adoc#install-rhacs-cloud-other[Setting up {product-title-managed-short} on other platforms]
+
+|Google Kubernetes Engine (Google GKE)
+|Limited ^[4]^
+|Yes
+
+|Microsoft Azure Kubernetes Service (Microsoft AKS)
+|Limited ^[4]^
+|Yes
+
+.2+|Self-managed platform
+|Red Hat {ocp} ({ocp-short}) 4.x
+|Yes
+|Yes
+.2+d|Operator (recommended), Helm charts, or roxctl CLI ^[3]^
+.2+a| * xref:../installing/installing_ocp/install-rhacs-ocp.adoc#install-rhacs-ocp[High-level overview of installing {product-title-short} on {osp}]
+* xref:../installing/installing_cloud_ocp/install-rhacs-cloud-ocp.adoc#install-rhacs-cloud-ocp[Setting up {product-title-managed-short} on {osp}]
+
+|{osp} Kubernetes Engine (OKE) 4.x
+|No
+|Yes
+|===
+
+[.small]
+--
+1. The availability of support for each platform depends on the overarching lifecycle of the platform and the end-of-life date.
+2. {rh-rhacs-first} is supported as a self-managed add-on for managed OpenShift services that meet the installation and sizing requirements for {product-title-short}. Deploying and managing {product-title-short} components does not fall under the service description for the managed service, and therefore {product-title-short} is not included in the SRE service.
+3. Do not use the `roxctl` installation method unless you have a specific installation that requires this method.
+4. {product-title-short} Central is tested, qualified, and is fully supported exclusively on {ocp} 4. Deployment and use of Central in environments that are not {ocp} 4 is possible, but support is limited to the {product-title-short} product software only and not to the underlying infrastructure provider. As part of problem diagnosis and isolation, it is necessary to reproduce problems in an {ocp} 4 environment. If an issue is specific to a provider and cluster that is not {ocp} 4, Red Hat provides commercially reasonable support to isolate issues. You are expected to open a case with your respective provider. For instructions, see the https://access.redhat.com/third-party-software-support[Red Hat 3rd Party Support Policy].
+--
+
+include::modules/installation-methods-for-different-architectures.adoc[leveloffset=+1]
+
+include::modules/supported-browsers-for-rhacs.adoc[leveloffset=+1]
+
+
+
+//content with footnotes
+////
+.Supported platforms and recommended installation methods
+[%autowidth]
+|===
+|*Platform type footnote:platform[The availability of support for each platform depends on the overarching lifecycle of the platform and the end-of-life date.]*|*Platform*|*Supported for Central*|*Supported for Secured Clusters*|*Supported installation methods*|*Installation steps*
```

Comment:
I think the preview looks good now without the footnotes. Feel free to delete the commented content.

Reply:
Done